packages/hydrojudge/src/sandbox.ts

```diff
@@ -230,7 +230,8 @@ export async function versionCheck(reportWarn: (str: string) => void, reportErro
         const config = await client.config();
         sandboxCgroup = config.runnerConfig?.cgroupType || 0;
     } catch (e) {
-        reportError('Your sandbox version is tooooooo low! Please upgrade!');
+        if (e?.code === 'ECONNREFUSED') reportError('Connecting to sandbox failed, please check sandbox_host config and if your sandbox is running.');
```

Comment:
ECONNRESET/ECONNTIMEOUT（？

maybe check by ret status or similar would be better

Reply:
I 'll treat all [system error](https://nodejs.org/api/errors.html#class-systemerror) during `connect` as connection failure.